packaging/hudi-flink-bundle/pom.xml

```diff
@@ -36,6 +36,7 @@
     <javax.servlet.version>3.1.0</javax.servlet.version>
     <!-- override to be same with flink 1.12.2 -->
     <parquet.version>1.11.1</parquet.version>
+    <hive.version>2.3.1</hive.version>
```

Comment:
@swuferhong thanks for your contribution. What issue does this PR fix? Would you add some description in this PR or in the JIRA ticket?

Reply:
This PR is to further support the Flink writer hive3 metastore synchronization and fix the kryo package conflict problem, as a follow-up PR of HUDI-1871  https://github.com/apache/hudi/pull/2968, which support Flink writer hive2 meta sync. And I will add some description. Thank you.